.eslintrc.cjs

```diff
@@ -15,9 +15,11 @@ const config = {
     },
   ],
   parser: "@typescript-eslint/parser",
-  parserOptions: {
-    project: path.join(__dirname, "tsconfig.json"),
-  },
+ project: path.join(__dirname, "tsconfig.json"),
+      },
+    },
+  ],
+  ignorePatterns: ["*.config.js"],
```

Comment:
I'm sorry but I meant the project property should be inside of parserOptions
```suggestion
parserOptions: {
        project: path.join(__dirname, "tsconfig.json"),
      },
    },
  ],
  ignorePatterns: ["*.config.js"],
```

Reply:
@Lada496 Oops! It's my bad. I just misunderstood that your previous suggestion contained project property inside parseOptions object